binscripts/rvm-installer

```diff
@@ -783,27 +783,33 @@ follow this link for details how to fix:
 
 rvm_install_validate_volume_mount_mode()
 {
-  \typeset path partition
+  \typeset path partition test_exec
 
   path=$rvm_path
 
-  # $rvm_path might not exists at this point so we need to traverse the directory tree upwards
-  # to find the name of the partition and then check if it has not been mount with noexec mode
+  # Directory $rvm_path might not exists at this point so we need to traverse the tree upwards
   while [[ -n "$path" ]]
```

Comment:
Yeah, I missed the comment, sorry for that. Me, personally, I'd create a script in the final location (~/.rvm/bin or similar, creating directories as required) but I didn't want to mess with the internals too much (maybe something later checks if the directory exists and would get confused).

Then we could also use a hardcoded script name and not rely on `mktemp` (which should be pretty universal but given you check for `grep` and `which` I understand you want minimal external dependencies)